scripts/gitlab-build.sh

```diff
@@ -192,47 +255,54 @@ push_binaries () {
   aws s3api put-object --bucket $S3_BUCKET --key $CI_BUILD_REF_NAME/$BUILD_PLATFORM/"parity_"$VER"_"$IDENT"_"$ARC"."$EXT".md5" --body "parity_"$VER"_"$IDENT"_"$ARC"."$EXT".md5"
   aws s3api put-object --bucket $S3_BUCKET --key $CI_BUILD_REF_NAME/$BUILD_PLATFORM/"parity_"$VER"_"$IDENT"_"$ARC"."$EXT".sha256" --body "parity_"$VER"_"$IDENT"_"$ARC"."$EXT".sha256"
 }
-make_archive () {
-  echo "add artifacts to archive"
-  rm -rf parity.zip
-  zip -r parity.zip target/$PLATFORM/release/parity$S3WIN target/$PLATFORM/release/parity-evm$S3WIN target/$PLATFORM/release/ethstore$S3WIN target/$PLATFORM/release/ethkey$S3WIN parity$S3WIN.md5 parity-evm$S3WIN.md5 ethstore$S3WIN.md5 ethkey$S3WIN.md5 parity$S3WIN.sha256 parity-evm$S3WIN.sha256 ethstore$S3WIN.sha256 ethkey$S3WIN.sha256
+prepare_artifacts () {
+  echo "__________Copy all artifacts to one place__________"
+  rm -rf artifacts
+  mkdir -p artifacts
+  cp target/$PLATFORM/release/parity$S3WIN artifacts
+  cp target/$PLATFORM/release/parity-evm$S3WIN artifacts
+  cp target/$PLATFORM/release/ethstore$S3WIN artifacts
+  cp target/$PLATFORM/release/ethkey$S3WIN parity$S3WIN.md5 artifacts
+  cp parity-evm$S3WIN.md5 ethstore$S3WIN.md5 artifacts
+  cp ethkey$S3WIN.md5 artifacts
+  cp parity$S3WIN.sha256 artifacts
+  cp parity-evm$S3WIN.sha256 artifacts
+  cp ethstore$S3WIN.sha256 artifacts
+  cp ethkey$S3WIN.sha256 artifacts
 }
-
 updater_push_release () {
-  echo "push release"
-
+  echo "__________Build comlete__________"
+  if [[ -z "$protect" ]]; then echo "__________Skipping push release__________"&&return; fi
+  echo "__________Push release___________"
   DATA="commit=$CI_BUILD_REF&sha3=$SHA3&filename=parity$S3WIN&secret=$RELEASES_SECRET"
   # Mainnet
   source scripts/safe_curl.sh $DATA "http://update.parity.io:1337/push-build/$CI_BUILD_REF_NAME/$BUILD_PLATFORM"
   # Kovan
   source scripts/safe_curl.sh $DATA "http://update.parity.io:1338/push-build/$CI_BUILD_REF_NAME/$BUILD_PLATFORM"
 }
-
+build_and_push () {
+  build
+  strip_binaries
+  calculate_checksums
+  make_deb
+  prepare_artifacts
+  push_binaries
+  updater_push_release
+}
 case $BUILD_PLATFORM in
   x86_64-unknown-linux-gnu)
     #set strip bin
     STRIP_BIN="strip"
     #package extention
     EXT="deb"
-    build
-    strip_binaries
-    calculate_checksums
-    make_deb
-    make_archive
-    push_binaries
-    updater_push_release
+    build_and_push
     ;;
   x86_64-unknown-debian-gnu)
     STRIP_BIN="strip"
     EXT="deb"
     LIBSSL="libssl1.1 (>=1.1.0)"
-    echo "Use libssl1.1 (>=1.1.0) for Debian builds"
-    build
-    strip_binaries
-    calculate_checksums
-    make_deb
-    make_archive
-    push_binaries
+    echo "__________Use libssl1.1 (>=1.1.0) for Debian builds__________"
+    build_and_push
```

Comment:
This changes the behaviour since we are pushing to auto-updater as well. Right now auto-updater is smart enough to reject such request (if the PLATFORM is set correctly). Although I think it would be good to avoid pushing to auto-updater if we know it's gonna be rejected (i.e. we only care about darwin, linux and windows packages).